web/src/scenes/app/Configuration.tsx

```diff
@@ -35,10 +72,23 @@ export const Configuration = memo(function Configuration() {
   const { updateAppName, updateAppDescription } = useApps();
   const [appName, setAppName] = useState<string>(initialValues.name);
   const [debouncedAppName] = useDebounce(appName, 1000);
```

Comment:
@michaelfurutin Let's remove debounced inputs and related logic which saves data on input. Just leave saving onBlur and saving using Enter button.